vault/data_source_policy_document.go

```diff
@@ -276,6 +289,10 @@ func policyRenderPolicyRule(rule *PolicyRule) string {
 		renderedRule = fmt.Sprintf("%s  required_parameters = %s\n", renderedRule, policyRenderListOfStrings(rule.RequiredParameters))
 	}
 
+	if rule.SubscribeEventTypes != nil {
```

Comment:
nit: I wonder if checking the length of the member would be clearer.

Reply:
Makes sense. Updated here: https://github.com/hashicorp/terraform-provider-vault/pull/2445/commits/c59531b10641f227c5a58e41a1c4f15d5438ddcc